manifests/p/ParallelsInternationalGmbH/ParallelsClient/20.1.25631/ParallelsInternationalGmbH.ParallelsClient.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: ParallelsInternationalGmbH.ParallelsClient

```

```suggestion
PackageIdentifier: Parallels.Parallels
```

Please update file name and path to match this change.